app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -2,27 +2,35 @@
 <%= bootstrap_form_for pregnancy do |f| %>
 
   <%= f.fields_for patient do |p| %>
-    <div class="col-md-2">
-      <%= p.text_field :primary_phone, label: "Phone Number" %>
+
+
+<div class="col-md-3">
```

Comment:
This is totally a trivial thing, but can we fix up the indentation here? We're almost certain to revisit this code once we get clarity from the designers, and right now it's kinda challenging to read at first glance. In particular, let's make sure the nested fields are indented in the block.